### PR TITLE
Card

### PR DESCRIPTION
**Pull request by manish-GH**
Added Card component and changed some of the variable names

Preview: https://elegant-shaw-deddd8.netlify.app/components/card/

### CODE REVIEW

File: components/Button/styles.css [hunk 1]
```diff
@@ -61,7 +61,7 @@
 	position: fixed;
 }
 .btn-floating:hover {
-	background-color: var(--theme-primary-light);
+	background-color: var(--theme-primary-400);
 }
 .btm-r {
```
Comment (JuvitaSaini09) [inline components/Button/styles.css:66]: ig class name should be btn-r

**Review by JuvitaSaini09 — APPROVED** [review]

File: components/Button/styles.css (hunk 1 above)
Comment (manish-GH) [reply components/Button/styles.css:66]: Actually it's correct  btm-r =>bottom right

**Comment by yaswanthmyneni** [comment]
everything looks good to me, can be merged.

**Review by nikhiljugale007 — COMMENTED** [review]
Code look good. 
One UI suggestion, for text-only card you add some padding around the text.

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, 
just suggestion  on the text only cards it would look nice if you can add some padding and also if text would be center.

**Review by mehulsatardekar — APPROVED** [review]
Just a suggestion, for a **card with a badge section** adding some padding between texts and button would look better, apart from that UI looks good.👌

**Pull request merged**